src/styles/blocks/collection.scss

```diff
@@ -0,0 +1,55 @@
+.collection {
+
+  &__photos {
+    margin-bottom: $margin-bottom-primary;
+  }
+
+  &__photo-container {
+    @extend %photoContainer;
+
+    max-height: $image-height-primary;
+
+    @include onTablet {
+      height: 180px;
+    }
+
+    @include onDesktop {
+      height: $image-height-primary;
+    }
+
+    &--hidden {
+      @include onTablet {
+        height: 285px;
+      }
+
+      @include onDesktop {
+        height: $image-height-primary;
+      }
+    }
+  }
+
+  &__image {
+    @extend %imageStyle;
+
+    object-position: 20% 50%;
+  }
+
+  &__info {
+    color: $c--red;
+
+    @include linkOnHover;
+
+    &--black {
+      font-weight: 400;
+      color: $c--black;
+
+      &:hover {
+        color: inherit;
+      }
```

Comment:
Useless

Reply:
![image](https://user-images.githubusercontent.com/121725856/218306157-3ec6036b-eb63-4994-98b9-b46fed28dc80.png)

I just wanted to exclude the date become red on hover

Reply:
So, don't make it a part of a hoverable element. You have a link for this. Make the paragraph a separate element.